.github/workflows/e2e.yml

```diff
@@ -135,3 +134,21 @@ jobs:
         if: always()
         run: |
           pip cache remove llama_cpp_python
+
+      - name: Add comment to PR if the workflow failed
+        if: failure() && steps.check_pr.outputs.is_pr == 'true'
+        run: |
+          gh pr comment "${{ github.event.inputs.pr_or_branch }}" -b "e2e workflow failed on this
+          PR: [View run](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{
+          github.run_id }}), please investigate."
```

Comment:
I think this might be treated as 3 independent lines. Maybe it'll work if you put `\` on the first two?  Or just make it one line.

or do you know this will work as intended?

Reply:
I also had doubts, let me change it.